src/protocols/vnc/settings.c

```diff
@@ -373,6 +374,8 @@ enum VNC_ARGS_IDX {
      */
     IDX_WOL_WAIT_TIME,
 
+    IDX_DISABLE_REMOTE_INPUT,
```

Comment:
Please document.

Reply:
Added documentation: "Whether or not to disable the input on the server side.".